.gitmodules

```diff
@@ -25,7 +25,7 @@
 	url = https://code.google.com/p/nanopb/
 [submodule "src/libs/openxc-message-format"]
 	path = src/libs/openxc-message-format
-	url = https://github.com/openxc/openxc-message-format
+	url = https://github.com/mgiannikouris/openxc-message-format
```

Comment:
I issued a pull request for the message format as well. The new version doesn't exist yet in openxc. Do you have a preference for the sequence of these with the dependency?